timeseries/src/autogluon/timeseries/models/ensemble/weighted/per_item_greedy.py

```diff
@@ -0,0 +1,138 @@
+import logging
+from typing import Any, Optional
+
+import numpy as np
+import pandas as pd
+from joblib import Parallel, delayed
+
+from autogluon.timeseries import TimeSeriesDataFrame
+from autogluon.timeseries.models.local.abstract_local_model import AG_DEFAULT_N_JOBS
```

Comment:
this looks a bit dangerous. should `AG_DEFAULT_N_JOBS` go to some higher level config/constants file?